cid-redirects.json

```diff
@@ -2265,7 +2265,6 @@
   "/Cloud_SIEM_Enterprise/CSE_Rules/Normalized_Threat_Rules": "/docs/cse/rules/normalized-threat-rules",
   "/Cloud_SIEM_Enterprise/CSE_Rules/Rule_Tuning_Expressions": "/docs/cse/rules/rule-tuning-expressions",
   "/Cloud_SIEM_Enterprise/CSE_Rules/Tailor_a_Global_Rule": "/docs/cse/rules/tailor-global-rule",
-  "/docs/cse/rules/first-seen-rule": "/docs/cse/rules/write-first-seen-rule",
```

Comment:
This was a 301 redirect I recently added to fix a high-traffic 404. Removing it will break the link again. Did someone ask to remove it?

Reply:
Oh, I'm sorry, I didn't know that. I had no idea you had made this change. That had to be removed because it was showing up in the current branch while compiling and causing build errors. This appeared to be an old cid redirect path to me for the first-seen rule that came with the other changes of the first-seen rules file.  However, I believe the change will be present in the main branch until this one is merged.
I was not merging this PR until those changes were made again.

Reply:
ok cool thank you for the update!